### ISSUE TITLE
$PERL5LIB breaks install

### ISSUE THREAD

**Issue by coderanger** [chef/chef-server#55]
If you, for reasons lost to the mists of time, have `$PERL5LIB` set and pointed at system-level Perl lib folders, the embedded perl in the installer will use those instead of the bundled ones. This then breaks sqitch with an error like:

```
/opt/opscode/embedded/bin/perl: symbol lookup error: /usr/lib64/perl5/auto/Fcntl/Fcntl.so: undefined symbol: Perl_Istack_sp_ptr
```

Might be good to appbundler-style it or just have the installer script specifically unset that var in its environment before running perl code.

**Comment by jjdevbiz** [comment]
For solidarities sake, heres what the full error looked like: https://gist.github.com/jjdevbiz/d22d33b99182d7be8cb6

**Comment by stevendanna** [comment]
Quick fix here: https://github.com/opscode/opscode-omnibus/pull/661

**Comment by stevendanna** [comment]
Fixed in 12.0.3
